sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -255,6 +261,7 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
       url,
       method: "GET",
       body: options?.body,
+      allowInsecureConnection: this.allowInsecureConnection,
```

Comment:
Right, all MI network requests happen on http

But, now it's in one place instead of multiple:

- https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/src/credentials/managedIdentityCredential/imdsMsi.ts#L177
- https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/src/credentials/managedIdentityCredential/appServiceMsi2019.ts#L96
- https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/src/credentials/managedIdentityCredential/cloudShellMsi.ts#L102-L103

Reply:
What is the default value of this field when not set?

Reply:
You can control+click your way into core-rest-pipeline, there you will see that it defaults to `false` https://github.com/azure/azure-sdk-for-js/blob/main/sdk/core/core-rest-pipeline/src/pipelineRequest.ts#L158 👍